content/docs/04.5-devx/03-cluster-groups.md

```diff
@@ -101,6 +101,35 @@ To delete a cluster group, select `Delete Cluster form the **Settings** and ente
 * Azure
 * AKS
 * VMWare
+* OpenShift
+
+<Tabs>
+<Tabs.TabPane tab="Configure OpenShift" key="Configure OpenShift">
+
+#### Configure OpenShift 
+<br />
+To deploy a new virtual cluster on OpenShift:
+
+
+1. Select the new Cluster Group and click **Settings**.
+2. Select **Settings** in **Cluster Group Settings**.
+3. In the **Advanced Config** file, locate the section ``Configure SecurityContext of the containers in the VCluster pod``.
+
+4. Comment out these lines: 
+
+    * ``fsgroup``
```

Comment:
It wouldn't hurt to be 100% explicit, e.g.:
`fsGroup: 12345`
`runAsGroup: 12345`
`runAsUser: 12345`

Reply:
@TylerGillson That's strange, the doc source file doesn't specify 12345 at all. It just lists the parameter names - not sure where it's picking up 12345. I've added the value shown in the image: 1001100000

Reply:
The defaults that we set are all `12345`

Reply:
`1001100000` came from my screenshot in Jira, which is not reflective of the correct default values.

Reply:
@TylerGillson I tried creating a new cluster group, and the default is 12345. But since this is for new and existing cluster groups, I really don't want to include a value. The doc just says to comment out those lines.